scripts/contracts-sync.sh

```diff
@@ -0,0 +1,9 @@
+# !/bin/bash
+
+# Pull lsp-smart-contracts repo
+mkdir tmpDocsSync 
+cd tmpDocsSync
+git clone --depth 1  --branch test-docs-sync https://github.com/lukso-network/lsp-smart-contracts.git
```

Comment:
@Hugoo @CallumGrindle this will be changed to `--branch main` once we are sure the script works and fetch our contracts docs correctly.

So it will then be set to `main` so that it updates the docs after each new release.